public class TryCatchExample {
    public static void main(String[] args) {
        try{
        int[] i={4,5,1};
        System.out.println(i[0]);
        }
        catch(Exception e)
        {
            
        System.out.println("An eception happened");
        }
            }
}
